test/lit/passes/experimental-new_eh.wast

```diff
@@ -0,0 +1,29 @@
+;; When given alone, --experimental-new-eh just runs --translate-to-new-eh
+;; RUN: wasm-opt %s -all --translate-to-new-eh -S -o %a.wasm
+;; RUN: wasm-opt %s -all --experimental-new-eh -S -o %b.wasm
+;; RUN: diff %a.wasm %b.wasm
+
+;; When given with other flags, --experimental-new-eh runs the translator after
+;; running other passes. If --optimize-level >=3, --experimenal-new-eh also runs
+;; StackIR (+ local2stack) optimization. So running '-O --experimental-new-eh'
+;; should be the same as running all these passes separately.
+;; RUN: wasm-opt %s -all -O --translate-to-new-eh -o %t.wasm
+;; RUN: wasm-opt %t.wasm -all --optimize-level=3 --generate-stack-ir --optimize-stack-ir -o %a.wasm
```

Comment:
Can we do without the intermediate file here? (@kripken, I don't quite remember how the optimization-level and -O flags interact these days)

Reply:
I think we can do without it... Not sure why I thought otherwise. Will fix.

Reply:
Removed the intermediate file. Will merge this. Thanks!